docs/en/observability/monitor-k8s/monitor-k8s-add-integration.asciidoc

```diff
@@ -0,0 +1,380 @@
+[discrete]
+[[monitor-kubernetes-integration]]
+== Part 1: Add and Configure the Kubernetes integration
+
+To start collecting logs and metrics from your Kubernetes clusters, first add the {integrations-docs}/kubernetes[Kubernetes integration] to your policy and configure the metrics and logs you want to collect.
+
+[discrete]
+[[monitor-k8s-add-integration]]
+=== Step 1: Add the Kubernetes integration to your deployment
+
+Follow these steps to add the Kubernetes integration to your policy:
+//add screenshots if necessary
+
+. Select *Add integrations* from your deployment's homepage.
++
+[role="screenshot"]
+image::images/add-integration.png[Add integration button]
+. Enter "Kubernetes" in the search bar, and select the *Kubernetes* integration.
+. Click *Add Kubernetes* at the top of the Kubernetes integration page.
+. Click *Add integration only (skip agent installation)* at the bottom of the Add integration page.
++
+[role="screenshot"]
+image::images/skip-agent-button.png[Add integration only (skip agent installation) button]
```

Comment:
I'm okay with leaving these as-is, but I do wonder if these screenshots are necessary.